### PR TITLE
Add warning in catalogue sharing

### LINKED ISSUE #1367: Missing list of catalog items that will not be shared
I find it very easy to miss the warning that you actually will not be sharing your data here:

image

Can we make it stand out more, perhaps show it in red? I'd also suggest adding some space before the dot point.

### PR DESCRIPTION
**Pull request by reginapramesti**
Fixes #1367

### CODE REVIEW

**Comment by reginapramesti** [comment]
Some things to note:

* The list of CSVs that won't be shared currently just lists a bunch of Unnamed Items. This should in theory be fixed by fixing #3399
* Clicking `Remove` from the workbench doesn't get rid of the warning as it doesn't actually remove the data from the catalogue but simply removes it from the map. Though removing it from the `My Data` tab would get rid of the warning

**Comment by tephenavies** [comment]
Please run `gulp lint` and fix the lint issues.

File: lib/ReactViews/Map/Panels/SharePanel/share-panel.scss [hunk 1]
```diff
@@ -124,3 +124,23 @@
...
+  padding: $padding;
+
+  ul {
+    padding-top: $padding;
+    padding-inline-start: 0;
+    list-style: none;
+  }
+
+  a {
+    color: #552800 !important;
```
Comment (tephenavies) [inline lib/ReactViews/Map/Panels/SharePanel/share-panel.scss]: Why do these have to be `!important`? Is there any way to avoid it?

**Comment by reginapramesti** [comment]
@steve9164 could you have a look at it now please?

**Review by tephenavies — APPROVED** [review]

**Pull request merged**